src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -150,6 +150,12 @@
             <argument type="service" id="sylius.variant_resolver.default" />
         </service>
 
+        <service id="sylius.order_processing.order_exchange_rate_and_currency_updater" class="Sylius\Component\Core\OrderProcessing\OrderExchangeRateAndCurrencyUpdater">
```

Comment:
Did we resign from class parameters at all? Nevertheless, rest of class definitions are placed in parameters, so we could keep consistency in a single file at least.